source/fab/steps/analyse.py

```diff
@@ -292,29 +292,31 @@ def _add_manual_results(special_measure_analysis_results, analysed_files: Set[An
         logger.info(f'added {len(special_measure_analysis_results)} manual analysis results')
 
 
-def _gen_symbol_table(analysed_files: Iterable[AnalysedDependent]) -> Dict[str, Path]:
+def _gen_symbol_table(analysed_files: Iterable[AnalysedDependent], debug=False) -> Dict[str, Path]:
```

Comment:
I don't understand the value of the `debug` argument. It seems to provide optional warnings when duplicates found. Much better would be to use the logging framework we already have to log the warnings. Then the user decides whether they see them in the usual manner.